argilla/tests/unit/test_record_ingestion.py

```diff
@@ -50,6 +51,19 @@ def test_ingest_record_from_dict(dataset):
     assert record.suggestions[0].value == "positive"
 
 
+def test_ingest_record_from_empty_dict_raises(dataset):
+    with pytest.raises(RecordsIngestionError):
+        dataset.records._ingest_records(
+            records=[
+                {
+                    "prompt": "What is the capital of France?",
+                    "label": "positive",
+                },
+                {},
```

Comment:
This should also fail:
```suggestion
                {},
                { "id": "record_id" },
```

Reply:
Yes, but line 62 will fail first. I've added separate raises contexts for each case.

Reply:
ahh sorry, I thought it was a test parameter.